Gemfile

```diff
@@ -1,5 +1,9 @@
 source 'https://rubygems.org'
 
+%w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
+end
```

Comment:
Hi @packrat386, I finally get read of this code. Sorry I get confused about your PR. I decided to take out your commits from here and then include your commits and rebase mines when your PR is accepted.
So, waiting for the final merging or CR. of #119 .
Looking forward. Cheers!